examples/Graphene_with_GPAW/run_and_analyze.py

```
@@ -0,0 +1,104 @@
+base_dir = os.path.dirname(os.path.abspath(__file__))
+bulk_dir = os.path.join(base_dir, "calculation_bulk")
+CORES = 8 # Set number of cores for MPI
+
+def run_calculation(folder):
+    """Runs the relax.py script in the given folder."""
+    print(f"Processing {folder}...")
+    relax_script = os.path.join(folder, "relax.py")
+    
+    # Check if already run (relaxed.gpw exists)
+    if os.path.exists(os.path.join(folder, "relaxed.gpw")):
+        print(f"  relaxed.gpw exists, skipping calculation.")
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)